src/pages/Main/main.scss

```
@@ -0,0 +1,218 @@
+.main {
+  .mainSliderContainer {
+    .mainSliderImage {
+      position: relative;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      width: 100%;
+      height: 100%;
+      transition: opacity 0.25s 0.25s ease-in;
+      opacity: 0;
+
+      .image {
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 550px;
+        object-fit: cover;
+      }
+
+      .mainSliderText {
+        position: absolute;
+        top: 100px;
+        left: 150px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: flex-start;
+        gap: 30px;
+        color: white;
+
+        .sliderSubTitle {
+          font-size: 15px;
+        }
+
+        .sliderTitle {
+          font-size: 85px;
+          font-weight: 600;
+        }
+
+        .shopNowButton {
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          font-weight: 500;
+          padding: 10px;
+          width: 110px;
+          height: 40px;
+          border: 1.5px solid #000;
+          border-color: white;
+        }
+      }
+    }
+
+    .active {
+      opacity: 1;
+    }
+
+    .mainSliderRightArrow {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      position: absolute;
+      top: 0;
+      right: 0;
+      width: 100px;
+      height: 550px;
+
+      i {
+        font-size: 50px;
+        color: white;
+      }
+    }
+
+    .mainSliderLeftArrow {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      position: absolute;
+      top: 0;
+      left: 0;
+      width: 100px;
+      height: 550px;
+
+      i {
+        font-size: 50px;
+        color: white;
+      }
+    }
+  }
+  .mainContainer {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100%;
+
+    .upperMargin {
+      margin-top: 550px;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+
+      .mainBestList {
+        padding: 75px 0;
+        max-width: 1260px;
+        .bestListTitleContainer {
+          position: relative;
+          display: flex;
+          justify-content: center;
+          align-items: center;
+          margin-bottom: 10px;
```

```suggestion
      .mainBestList {
        padding: 75px 0;
        max-width: 1260px;
        
        .bestListTitleContainer {
          position: relative;
          display: flex;
          justify-content: center;
          align-items: center;
          margin-bottom: 10px;
```
선택자 사이는 가독성을 위해서 한칸씩 띄워주세요!